themes/bootstrap3/templates/search/list-grid.phtml

```diff
@@ -1,4 +1,4 @@
-<div class="search-grid">
+<div class="search-grid <?=$this->escapeHtmlAttr($this->resultsClass ?? '')?>">
```

Comment:
Should we do something like:

`class="<?=$this->escapeHtmlAttr(trim('search-grid ' . ($this->resultsClass ?? ''))?>"`

...to avoid empty whitespace at the end of the HTML attribute?